reference-manual/src/main/java/MLModelPrediction.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.jet.Jet;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.ServiceFactory;
+import com.hazelcast.jet.pipeline.StreamStage;
+
+import hex.genmodel.MojoModel;
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowData;
+import hex.genmodel.easy.prediction.BinomialModelPrediction;;
+
+public class MLModelPrediction {
+    static void s1() {
+        //tag::s1[]
+        String modelFileLocation = "/path/to/gbm_model.zip";
+        JetInstance jet = Jet.newJetInstance();
+
+        JobConfig jobConfig = new JobConfig();
+        jobConfig.setName("H20 Classification Example");
+        jobConfig.attachFile(modelFileLocation, "gbm_mojo");
+        Job job = jet.newJob(buildPipeline(), jobConfig);
+        //end::s1[]
+    }
+
+    static void s2() {
+        //tag::s2[]
+        StreamStage<ItemToBeClassified> sourceStage = sourceStage();
+        sourceStage.mapUsingService(ServiceFactory.withCreateContextFn(context -> {
+            String model = context.attachedFile("gbm_mojo").toString();
+            return new EasyPredictModelWrapper(MojoModel.load(model));
+        }).withCreateServiceFn((context, easyPredictModelWrapper) -> easyPredictModelWrapper), 
```

Comment:
You can use `ServiceFactories.sharedService` and you won't need the `withCreateServiceFn`